website/blog/2020-07-14-lithium.md

```diff
@@ -0,0 +1,116 @@
+---
+author: Tomasz Godzik
+title: Metals v0.9.2 - Lithium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.9.2, which adds the support for
+Scala 2.12.12.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">41</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">12</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">6</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">14</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">1</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/25?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Scala 2.12.12 support
+
+## Miscellaneous improvements
+
+- Named parameters completions fixed to work in cases where the method was
+  defined later in the class.
+- Add option to not install and apply bloop in gradle builds.
+- Find all templates from giter8 wiki page for the New Project Provider
+
+## Contributors
+
+Big thanks to everybody who contributed to this release or reported an issue!
+
+```
+$ git shortlog -sn --no-merges v0.9.1..v0.9.2
+Chris Kipp
+Krzysztof Bochenek
+Tomasz Godzik
+Jorge Vicente Cantero
+Claudio Bley
+Robin Raju
+
+```
+
+## Merged PRs
+
+## [v0.9.2](https://github.com/scalameta/metals/tree/v0.9.2) (2020-07-14)
+
+[Full Changelog](https://github.com/scalameta/metals/compare/v0.9.1...v0.9.2)
+
+**Merged pull requests:**
```

Comment:
How come the Reset Notifications Popup pr that @kpbochenek did isn't in here? That's also a good one to mention up above since it's user facing.

Reply:
I am working on that, it seems my script for listing PRs was flaky. But I managed to ger it more bullet proof :smile:

Reply:
Fixed!